skin-resque-frontend/src/components/Arrow/Arrow.js

```diff
@@ -0,0 +1,24 @@
+import { FiChevronLeft, FiChevronRight } from 'react-icons/fi';
+import PropTypes from 'prop-types'
+
+
+function Arrow({left, right, onClick, color}) {
+  return (
+    <div onClick={onClick}>{left && !right ? <FiChevronLeft size={45} color={color}/> : <FiChevronRight size={45} color={color}/>}</div>
+  )
+}
+
+export default Arrow
+
+const requiredPropsCheck = (props, componentName) => {
```

Comment:
This check didn't go into effect when I was testing this component. With neither `left` or `right` it defaulted to right arrow. I would suggest switching to different library that throws error instead of logging it to console